### PR TITLE
Release 3.0.1

### PR DESCRIPTION
**Pull request by rebeccahum**
:warning: DO NOT MERGE (YET) :warning:

[Remaining work for this Milestone](https://github.com/Automattic/VIP-Coding-Standards/milestone/X)

PR for tracking changes for the X.Y.Z release. Target release date: DOW DD MMMM YYYY.

- [ ] Scan WordPress (or just wp-admin folder) with prior version and compare results against new release for potential new bugs.
- [ ] Add change log for this release: PR #XXX
- [ ] Double-check whether any dependencies need bumping.
- [ ] Merge this PR.
- [ ] Add signed release tag against `main`.
- [ ] Close the current milestone.
- [ ] Open a new milestone for the next release.
- [ ] If any open PRs/issues which were milestoned for this release do not make it into the release, update their milestone.
- [ ] Write a Lobby post.
- [ ] Write an internal P2 post.
- [ ] Open PR to update [Review Bot dependencies](https://github.com/Automattic/vip-go-ci/blob/master/tools-init.sh).

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,18 +17,18 @@
 	],
 	"require": {
 		"php": ">=5.4",
-		"phpcsstandards/phpcsextra": "^1.1.0",
-		"phpcsstandards/phpcsutils": "^1.0.8",
+		"phpcsstandards/phpcsextra": "^1.2.1",
+		"phpcsstandards/phpcsutils": "^1.0.9",
 		"sirbrillig/phpcs-variable-analysis": "^2.11.17",
-		"squizlabs/php_codesniffer": "^3.7.2",
+		"squizlabs/php_codesniffer": "^3.8.0",
```
[the remark this answers is not in the document]
Comment (jrfnl) [reply composer.json:23]: Definitely! Maybe even to `3.9.2` ?

Refs:
* https://github.com/PHPCSStandards/PHP_CodeSniffer/releases/tag/3.8.1
* https://github.com/PHPCSStandards/PHP_CodeSniffer/releases/tag/3.9.0
* https://github.com/PHPCSStandards/PHP_CodeSniffer/releases/tag/3.9.1
* https://github.com/PHPCSStandards/PHP_CodeSniffer/releases/tag/3.9.2

File: composer.json [hunk 2]
```diff
@@ -17,18 +17,18 @@
 	],
 	"require": {
 		"php": ">=5.4",
-		"phpcsstandards/phpcsextra": "^1.1.0",
-		"phpcsstandards/phpcsutils": "^1.0.8",
+		"phpcsstandards/phpcsextra": "^1.2.1",
+		"phpcsstandards/phpcsutils": "^1.0.9",
 		"sirbrillig/phpcs-variable-analysis": "^2.11.17",
```
Comment (jrfnl) [inline composer.json:22]: Also ?

```suggestion
		"sirbrillig/phpcs-variable-analysis": "^2.11.18",
```

File: composer.json [hunk 3]
```diff
@@ -17,18 +17,18 @@
...
-		"squizlabs/php_codesniffer": "^3.7.2",
+		"squizlabs/php_codesniffer": "^3.8.0",
 		"wp-coding-standards/wpcs": "^3.0"
```
Comment (jrfnl) [inline composer.json:24]: And:

```suggestion
		"wp-coding-standards/wpcs": "^3.1.0"
```

**Comment by jrfnl** [comment]
@rebeccahum If I look at the git graph, this branch looks wrong. Looks like everything has been cherrypicked from `develop` into a release branch.

If I look at the graph for previous releases, there have been one of two different workflows which look to have been followed:
1. pull changelog to `develop` and merge `develop` to `main` via a PR, then tag the `main` branch to release.
1. create a release branch based on `develop` which includes the changelog commit and pulled that branch to `main`, merge & tag `main` and once the release is out, `main` (or the release branch) should be merged to `develop` to get the changelog up to date in the `develop` branch.

The first workflow seems simplest to me.

Was there any particular reason not to do it that way this time ? Maybe time to document the release process properly and add a `release-checklist.md` file to the `.github` folder ? (unofficial convention)

**Comment by rebeccahum** [comment]
Okay, I'll re-make this PR then.

**Pull request closed without merging**